### ISSUE TITLE
The audio track disappears because of the reencode

### ISSUE THREAD

**Issue by LordBex** [crunchy-labs/crunchy-cli#47]
https://github.com/crunchy-labs/crunchy-cli/blob/027047fc7e22f5a9d439fd6fa4467ad00b3f3a3c/cli/commands/archive/archive.go#L664

I noticed that since the update with the reencode, when downloading several audio tracks, only the Japanese one was left
I therefore removed the code for reencode and tried again.
Now they were all there.

Mein Test Command war
```
./crunchy archive 'https://beta.crunchyroll.com/de/series/G1XHJV2X9/lycoris-recoil[S01E02]' --merge audio  -v --resolution worst -l ja-JP -l de-DE 
```

MediaInfo with reencode:
```
Matroska:97,4 MiB, 24 min 2s
1 Video stream: AVC
1 Audio stream: AAC LC                         <- different number
1 Text stream: ASS
Dauer: 24 min 2s
Gesamte Bitrate: 567 kb/s.                      <- and here a different bitrate 
Kodierendes Programm: Lavf59.27.100
verwendete Encoder-Bibliothek: Lavf59.27.100
ErrorDetectionType: Per level 1
```

without:
```
Matroska:109 MiB, 24 min 2s
1 Video stream: AVC
2 Audio streams: AAC LC / AAC LC.            <- different number
1 Text stream: ASS
Dauer: 24 min 2s
Gesamte Bitrate: 632 kb/s.                           <- and here a different bitrate 
Kodierendes Programm: Lavf59.27.100
verwendete Encoder-Bibliothek: Lavf59.27.100
ErrorDetectionType: Per level 1
```

**Comment by ByteDream** [comment]
I could swear in my test everything was fine. I'll look into it.

![jkoeiw12t8v41-2075333712](https://user-images.githubusercontent.com/63594396/188334615-411503f0-78db-43c7-87bb-09a2713a6648.jpg)

**Comment by ByteDream** [comment]
Fixed in release [v2.3.2](https://github.com/crunchy-labs/crunchy-cli/releases/tag/v2.3.2)🤞
